test/membrane_opus/encoder_test.exs

```diff
@@ -11,6 +11,24 @@ defmodule Membrane.Opus.Encoder.EncoderTest do
   @input_path "test/fixtures/raw_packets"
   @reference_path "test/fixtures/encoder_output_reference"
 
+  defp raw_packets(with_pts?) do
+    File.read!(@input_path)
+    |> String.split()
+    |> Enum.zip(0..2)
```

Comment:
```suggestion
    |> Stream.with_index(1)
```

I know that I suggested you to use `Enum.zip`, but now I think that `Stream.with_index` will be better, to cover the case when somebody will modify content inside `@input_path`